authentication/login.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://integral-ui-cl.netlify.app/CSS/allstyles.css">
+    <link rel="stylesheet" href="https://integral-ui-cl.netlify.app/CSS/nav.css">
+    <link rel="stylesheet" href="../style.css">
+    <script src="https://kit.fontawesome.com/0ce6b5a07c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+   <div class="container auth-container">
+
+            <div class="card auth-card">
+                <div class="card-title">
+                <h2 class="heading-2">Welcome Back</h2>
+                <span class="misc">
+                    New here? <a class="link blue" href="/authentication/signup.html">Create an account for free!</a>
```

Comment:
giving class as "misc" sounds odd.. because we give classes to indicate what it does, maybe you could change it something else? as it seems to center text , say text-centered or auth-subtext?